src/extension.ts

```diff
@@ -101,7 +102,10 @@ export function activate(context: vscode.ExtensionContext): void {
             addBlocknumbersCommand()
         ),
         vscode.commands.registerCommand("isg-cnc.StartDocu", () =>
-            startDocu()
+            openDocu()
+        ),
+        vscode.commands.registerCommand("isg-cnc.openDocuWithId", (id: string) =>
```

Comment:
Es gibt in der Doku eine Seite mit der Übersicht von allen # Befehlen (https://www.isg-stuttgart.de/fileadmin/kernel/kernel-html/de-DE/index.html#525201163). Könnten wir das nicht auch nutzen bei der Hilfeaufrufe bei markierten # Befehlen? Oder erkennen wir die bisher noch gar nicht

Reply:
Allgemein #-Befehle könnten wir wahrscheinlich recht unproblematisch erkennen und dann in die Übersicht sprigen.
Um dann in die Unterseiten der Übersicht (also konkret ein spezieller #-Befehl) zu springen bräuchten wir dann eine JSON ähnlich zur cycles.json. Das wäre natürlich umso besser, wenn wir das von der Doku bekommen könnten.

Reply:
Ok, ich frage mal nach, aber denke nicht, dass wir da so eine Datei bekommen werden und wir uns mit der Übersichtsseite begnügen müssen. Können wir aber auch in einen anderen Issue auslagern.

Reply:
Hab ich mal als eigenes [Issue](https://github.com/isg-stuttgart/vscode-isg-cnc/issues/155) aufgemacht.